code/modules/power/supermatter/supermatter.dm

```diff
@@ -341,26 +345,27 @@
 
 			if(damage > emergency_point)
 				SPEAK("[emergency_alert] Instability: [stability]%")
+				SPEAK("[emergency_alert] Instability: [stability]%", radio_channel)
```

Comment:
I know - I wasn't sure if I wanted to do it that way or not. It's easy to miss shit in the common channel sometimes so I figured the redundant message would be helpful for engineers, but it's nbd either way